### PR TITLE
Issue 77

### PR DESCRIPTION
**Pull request by mukoki**
This should partly fix issue 77. I will split the issue into several parts to track parts of the issue which have not yet been solved.

### CODE REVIEW

File: src/test/java/org/cts/parser/prj/PrjParserTest.java [hunk 1]
```diff
@@ -43,7 +47,7 @@
  *
  * @author Erwan Bocher
```
Comment (ebocher) [inline src/test/java/org/cts/parser/prj/PrjParserTest.java]: Please add @author @mukoki

**Comment by ebocher** [comment]
Don't forget to add your name as author.
By the way, a big thanks

**Comment by mukoki** [comment]
I added my name in modified files, let me know if the PR needs more change.

**Comment by ebocher** [comment]
Thanks a lot

**Pull request merged**